src/main/java/org/ssu/edu/teachua/ui/components/modal/add_club_component/AddClubContactsComponent.java

```diff
@@ -37,18 +38,21 @@ public AddClubContactsComponent(WebDriver driver) {
         super(driver);
     }
 
+    @Step("Open Add Location pop-up")
     public AddLocationComponent clickAddLocationButton() {
         waitForElementToBeClickable(addLocationButton);
         this.addLocationButton.click();
         return new AddLocationComponent(driver);
     }
 
+    @Step("Make club online")
     public AddClubContactsComponent clickOnlineSwitchButton() {
         waitForElementToBeClickable(onlineSwitchButton);
         this.onlineSwitchButton.click();
         return this;
     }
 
+    @Step("Enter Facebook contact")
     public AddClubContactsComponent enterContactFacebook(String contactFacebook) {
```

Comment:
add parameters to step ("Enter Facebook contact {contactFacebook}")

Reply:
you should review all steps below and add parameters where entering something or uploading photo